controllers/admin/AdminPsThemeCustoConfiguration.php

```diff
@@ -527,16 +527,7 @@ public function setFinalList($aList)
                 } else {
                     foreach ($aElementsList as $sModuleName => $iModuleId) {
                         if (!in_array($sModuleName, $modulesOnDisk)) {
-                            if ($this->getModule()->ready !== false) {
```

Comment:
`$this->getModule()->ready` what the fuck is that?

Could we add a hook at least? In order to be able to redownload or do something if the module is not on the disk?

Reply:
`$this->ready = (getenv('PLATEFORM') === 'PSREADY') ? true : false;` Oh shit !

Reply:
@PierreRambaud Done in #49 49

Reply:
You forget to read my second question little padawan :trollface:

Reply:
It's not on the scope of the PR. It should be done when an API will be available.